aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,50 @@
+
+        events.delete_rule(Name=rule_name)
+        print(f"EventBridge rule {rule_name} deleted.")
+
+        return {
+            "statusCode": 200,
+            "body": json.dumps(
+                f"Bucket {bucket_name} and all its objects deleted successfully. EventBridge rule removed."
+            ),
+        }
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)